RoboJackets/robocup-software
# Basic offensive indirect

## Description
A basic offensive indirect play

## Associated Issue
Issue #1408

## Steps to test
### Test Case 1
1. Setup an indirect in the offensive half of the field 
2. Run the play
3. Observe how well it works
4. Repeat with other areas on the field
